src/main/java/be/syntra/devshop/DevshopBack/services/CartService.java

```diff
@@ -1,9 +1,9 @@
 package be.syntra.devshop.DevshopBack.services;
 
-import be.syntra.devshop.DevshopBack.entities.Cart;
+import be.syntra.devshop.DevshopBack.entities.ShopOrder;
 
 
 public interface CartService {
```

Comment:
according to the conceptual logic, you should rename this service as well

Reply:
fixed